.github/workflows/ci.yml

```diff
@@ -11,82 +11,62 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Setup repo
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
 
       - name: Setup Deno
         uses: denoland/setup-deno@main
         with:
-          deno-version: 'v1.x'
+          deno-version: "v2.x"
 
       - name: Check Formatting
-        run: deno fmt --check --ignore=sqlite

```

Comment:
Why remove the --check? It should not format on CI but only check for formatting

Reply:
good catch, just a typo